### PR TITLE
fixed gatsby build error on 404, added storybook to readme

### PR DESCRIPTION
**Pull request by SammySteiner**

### CODE REVIEW

File: src/pages/404.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import Layout from '../components/layout/Layout';
 import Seo from '../components/seo/Seo';
+import { Content } from '../types/siteMetadata';
+
+import content from './index.content.yml';
 
 export default function NotFoundPage() {
   return (
-    <Layout>
+     <Layout content={content as Content}>
```
Comment (vonbearshark) [inline src/pages/404.tsx:10]: Hm. Wondering if we should either mark `content` as optional or import directly in `Layout`? Either way `404` shouldn't have to import `index.content.yml`

**Review by vonbearshark — COMMENTED** [review]
🤔 interesting `package-lock` diffs. Seem innocuous?

File: src/pages/404.tsx (hunk 1 above)
Comment (jessethach) [reply src/pages/404.tsx:10]: Optional makes sense to me.

**Review by jessethach — APPROVED** [review]
Thank you!

File: src/pages/404.tsx (hunk 1 above)
Comment (SammySteiner) [reply src/pages/404.tsx]: Sounds good. New to typescript, but I think I got it.

File: src/pages/404.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Seo from '../components/seo/Seo';
 
 export default function NotFoundPage() {
   return (
-    <Layout>
+     <Layout >
```
Comment (vonbearshark) [inline src/pages/404.tsx:7]: nit: weird spacing? Wonder why husky/prettier missed this

**Review by vonbearshark — APPROVED** [review]

File: src/components/layout/Layout.tsx [hunk 3]
```diff
@@ -6,14 +6,17 @@ import SEO from '../seo/Seo';
 
 type LayoutProps = {
   children: ReactNode;
-  content: Content;
+  content?: Content;
 };
 
 export default function Layout({ children, content }: LayoutProps) {
+
+  const title = content ? content.home.title : ''
```
Comment (vonbearshark) [inline src/components/layout/Layout.tsx:14]: nit: fwiw I am a big fan of reducing variables, [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). Making this
`<Header siteTitle={content?.home?.title ?? ''} />`
for example

File: src/pages/404.tsx (hunk 2 above)
Comment (SammySteiner) [reply src/pages/404.tsx]: I may not have noticed the `npm run format` until now. 😊

**Pull request merged**